### ISSUE TITLE
Feature-request: Chose base for jails

### ISSUE THREAD

**Issue by ntinti** [cbsd/cbsd#327]
It would be nice to have the possibility to select and change the base for jails at any time. E.g. to switch from FreeBSD.org base to self build base.tgz before or after a jail is constructed.
Currently  a selection only is shown before the build and only when a base isn't found

**Comment by olevole** [comment]
hint for myself: some kind of script like 'world-tui' or 'world-cfg' ?

**Comment by olevole** [comment]
notes: some of the improvements for this task were made as part of https://github.com/cbsd/cbsd/issues/367, now we can easy manage sources for bases via config files. 

so you can do 'cbsd removebase' , 'cbsd removebase ver=XX' and try to start jail again to get selector for base source.

anyway,  something like 'world-tui' would be nice to implement as well as '**basename**' in jconstruct-tui/jconfig. therefore, I will leave this task open and apparently it will be implemented after 12.0.4 release so that some testing time for https://github.com/cbsd/cbsd/issues/367 has passed.

**Comment by olevole** [comment]
Well, the ability to work with different bases was from the very first CBSD versions ( **basename**, see: https://www.bsdstore.ru/en/base_cbsd.html ). However, we lacked the usual management for this.

The 'basename' was returned to 'jconstruct-tui'. In addition, added *register_basename* args to 'installworld' and 'repo' script which governs the alternate base name obtained from the relevant sources. E.g:

```
cbsd repo action=get sources=base register_basename=version1
cbsd jset basename=version1 jname=XXX
```

or

```
cbsd buildworld
cbsd installworld register_basename=alternative_base
cbsd jset basename=alternative_base jname=YYY
```

or set basename in 'jconfig' or 'jconstruct-tui'.

Note: 'basename' menu item available only when you have named bases ( see: cbsd bases ).

Will be available in the next CBSD release: 12.0.14

**Comment by ntinti** [comment]
👍
